### ISSUE TITLE
Too much invalid shares

### ISSUE THREAD

**Issue by junforever** [djm34/ccminer-msvc2015#23]
Hello can anybody help me with this, according to the miner my accepted shared are 4131/4131 but in the dashboard of my account the invalid number is 47.62% of my shares, why the long difference? which is wrong? If the miner is wrong could you tell me please which are the possible causes of so many rejected percentage please? 
I have 3 gpus: gtx 1070  G1 rev1 2 gtx 1080 ti ftw3, my oc settings are: 99 core - 600 mem  (1070),  60 core 500 mem(1080), 85 core 500 mem (1080)  I am in windows 7 and my driver version of nvidia  is 385.69 I am using the ccminer_cuda9.exe  Thanks for the help
![valid](https://user-images.githubusercontent.com/10953490/32034387-f0bd270a-b9d7-11e7-8eb1-823220e6cb83.PNG)

**Comment by tinita2085** [comment]
Hi, I have the same issue but with less % of invalid shares, can someone tell us how to fix it. Thanks for your help

**Comment by chilak1l** [comment]
i try to increase the intensity -i 18 on my 1070s

**Comment by UselessGuru** [comment]
I can confirm that this miner produces quite a substantial number of invalid shares (testet with MiningPoolHub).
My cards: 1x GTX 1080ti, 1x GTX 1060 3GB, Driver Version 388.13
DJM34 Lyra2z results in 4.35kH/s, but approx 10% invalid shares
Ccminer 2.2.2 results in 4.2kH/s, but NO invalid shares
OC ist not relevant here.

**Comment by djm34** [comment]
It is indeed a known problem which was existing in the ccminer version I used to for implementing lyra2z

**Comment by tamlin-mike** [comment]
@djm34 Perhaps the following could help?
https://github.com/tpruvot/ccminer/search?q=stratum+stale+jobs+workaround

**Comment by hapklaar** [comment]
Are you going to build a new version without this problem? Like the extra hashrate, but reverted to tpruvot's latest version for now.

**Comment by djm34** [comment]
no, not for the moment. The stale share considerably reduce the number of valid share (at least on suprnova) and for that reason was remove from that version. 
The problem isn't originating from there. Now, you can probably try tpruvot version and see if it produce the same effect or not.
